.travis.yml

```diff
@@ -34,7 +34,7 @@ matrix:
 
       # Test Bazel build
       - rm -rf build
-      - ./ci/travis/install-bazel.sh
+      - out=`mktemp`; ./ci/travis/install-bazel.sh >$out 2>&1 || cat $out
```

Comment:
What is the purpose of this change? It doesn't make the log shorter, right?
Also, it'd a bit nicer to use a function. As this pattern is repeated many times.

Reply:
@raulchen here all output is suppressed unless the script exits with failure.

Agree about a function. I can make a script like run_silent.sh for other tests.

Reply:
> @raulchen here all output is suppressed unless the script exits with failure.

I see. This would be useful. However, there's a drawback of doing this. When a command hangs, there will be no output at all. And it's difficult to find out where the command is stuck at. Maybe in the `run_silent.sh` script, we can print the file if there's no output to the file for 5 minutes?

Also, another thing to notice is that travis will kill a job if there's no output for 10 minutes. We need to make sure this won't happen if a command simply needs more than 10 minutes.

Reply:
It turns out you can use `timeout` to enforce a time limit. I set it to 30 minutes here.

To avoid the travis timeout, we could also set it to 9 minutes, if we are sure each build step takes less than that time.

Reply:
Yeah, just saw you already did this.

Maybe we can print `Command xxx is still running.` every 9 minutes.